### PR TITLE
Series describe

### PR DESCRIPTION
**Pull request by r-brink**
Simple function to get summary statistics from a Series.

### CODE REVIEW

File: py-polars/tests/test_series.py [hunk 1]
```diff
@@ -279,3 +279,21 @@ def test_iter():
...
+
+    assert num_s.describe() == {
+        "min": 1,
+        "max": 3,
+        "sum": 6,
+        "mean": 2.0,
+        "std": 0.816496580927726,
+        "count": 3,
+    }
+    try:
```
Comment (PrettyWood) [inline py-polars/tests/test_series.py:296]: Probably better to write
```py
with pytest.raises(TypeError) as exc_info:
    str_s.describe()
assert str(exc_info) == "This type is not supported"
```

Comment (PrettyWood) [reply py-polars/tests/test_series.py:296]: And you probably need to do the same with empty serie and `ValueError`

Comment (r-brink) [reply py-polars/tests/test_series.py]: Thanks for the review and the code snippet! I have added the additional tests

**Comment by ritchie46** [comment]
LGTM. Thanks Robin!

**Pull request merged**